base/common/CMakeLists.txt

```diff
@@ -49,6 +47,17 @@ add_custom_command(
     COMMAND ${CMAKE_COMMAND} -E create_symlink ${COMMONS_NET_JAR} lib/commons-net.jar
 )
 
+if(WITH_JAXB)
+    add_custom_target(pki-jaxb-lib ALL
+        COMMENT "Creating links for JAXB library")
+
+    add_custom_command(
+        TARGET pki-jaxb-lib
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_API_JAR} lib/jaxb-api.jar
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_IMPL_JAR} lib/jaxb-impl.jar
```

Comment:
But re-add jakarta-activation here.

Reply:
You mean JDK11 doesn't need any of JAXB files?

Reply:
Hmm, not quite what I meant. I was thinking we needed:

```
        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_API_JAR} lib/jaxb-api.jar
        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_IMPL_JAR} lib/jaxb-impl.jar
        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAKARTA_ACTIVATION_JAR} lib/jakarta.activation.jar
```

i.e., all three. Because Jakarta-activation (under JDK8) duplicates JDK internals. The latter two were all added by the same PR, btw.